styleguide/_themes/derek/scss/globalElements/basement.scss

```diff
@@ -83,3 +89,59 @@
   float: left;
   font-size: 1.2rem;
 }
+
+.basement-iconList {
+  list-style: none;
+  margin-left: -5px;
+  padding-left: 0;
+}
+
+.basement-iconListItem {
+  float: left;
+}
+
+.basement-iconBg {
+  color: $white;
+  display: inline-block;
+  font: normal normal normal 1.83em FontAwesome;
```

Comment:
I had to copy over some stuff from font awesome for the 'fa' class since it was overriding font size. It looks like those were unnecessary so I removed them.